.eslintrc.js

```diff
@@ -5,20 +5,31 @@ module.exports = {
     browser: true,
     jest: true,
   },
-  extends: ['airbnb-base', 'plugin:flowtype/recommended', 'prettier', 'plugin:prettier/recommended'],
+  extends: [
+    'airbnb-typescript/base',
+    "airbnb/hooks",
```

Comment:
I guess we should use quotes consistently 😉

Reply:
My choice would have been for double quotes, but I tried to stick with the Prettier rules that were already present.

Reply:
I was just talking about some `extends` strings being single- and some being doublequoted. I would prefer enforcing consistent quotes for the whole project.

Reply:
Ah my bad.  Was reading that comment on mobile.  Not sure why I've got a mixture there.  Will fix.